core-processor/src/executor.rs

```diff
@@ -197,39 +299,27 @@ pub fn execute_wasm<A: ProcessorExt + EnvExt + IntoExtInfo + 'static, E: Environ
         settings.forbidden_funcs,
     );
 
-    let mut env = E::new(ext, program.raw_code(), &pages_data, mem_size).map_err(|err| {
-        log::error!("Setup instance err = {}", err);
+    let mut env = E::new(ext, program.raw_code(), mem_size).map_err(|err| {
+        log::debug!("Setup instance err = {}", err);
```

Comment:
```suggestion
        log::error!("Setup instance err = {}", err);
```
Error should be error

Reply:
It's not error for runtime, it's instantiation error which actually caused by wrong contract. I think we should use log::error for cases which happens unexpectedly for us.

Reply:
agree